HACKING.rst

```diff
@@ -0,0 +1,95 @@
+*********
+Craft Providers
+*********
```

Comment:
We should update `lint-docs` to also lint the rst files on the root I think (the over/underline here needs to have the same length as the title, although this is a minor nitpick)

Reply:
Great idea, I updated the call (and submitted a [feature request](https://github.com/sphinx-contrib/sphinx-lint/issues/66) upstream)